firebase/database.rules.json

```diff
@@ -55,6 +55,7 @@
         }
       },
       "users": {
+        ".read": true,
         "$uid": {
           ".read": "auth != null && auth.uid == $uid",
```

Comment:
we might need to adjust (or remove) this line as well, since it will overwrite the other rule two lines above. Otherwise the leaderboard would only be able to get the user ids and not the actual data

Reply:
double checked it. The upper statement overwrites the lower one. I will remove it, since it's a bit confusing to have it when it has not effect.

Reply:
I think it's fine but let me give it a shot

Reply:
sorry, my sentence was confusing.

Your statement is correct, but I will remove line 60. :)

Reply:
Maybe remove it for the v1 data as well? It has a similar structure at the moment @Hagellach37